.github/workflows/ci-join.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Run join test
         run: |
           cd scripts/join
-          go test . -v --integration --logs_file=docker_logs.txt
+          go test . -v --integration --logs_file=docker_logs.txt -timeout 8h
```

Comment:
I'd just disable timeout since the test handles timeouts itself, so ``--timeout=0`

Reply:
Ah thanks, yeah this is even better.